src/assignment/modals/ImportToAssignmentModal.tsx

```diff
@@ -103,20 +104,19 @@ const ImportToAssignmentModal: FunctionComponent<ImportToAssignmentModalProps &
 			);
 			const columnDataType = (column?.dataType ||
 				TableColumnDataType.string) as TableColumnDataType;
-			const assignmentData = await AssignmentService.fetchAssignments(
-				true, // canEditAssignments,
-				commonUser,
-				false, // not past deadline
+			const assignmentData = await AssignmentService.fetchAssignments({
+				canEditAssignments: true,
+				pastDeadline: false,
 				sortColumn,
 				sortOrder,
-				columnDataType,
-				0, // page
-				filterString, // filter,
-				[],
-				[],
-				[],
-				null // limit: no limit
-			);
+				tableColumnDataType: columnDataType,
+				offset: 0,
+				limit: ITEMS_PER_PAGE,
+				filterString,
+				labelIds: [],
+				classIds: [],
+				shareTypeIds: [],
```

Comment:
Still need to pass empty arrays here? Looks like they're set to default to `undefined` in the data call